tests/manage/test_pvc_disruptive.py

```diff
@@ -202,11 +198,6 @@ def disruptive_base(self, operation_to_disrupt, resource_to_delete):
             pvc_obj.delete()
```

Comment:
We'll need to find a way to utilize the factory mechanism in order to create the PVCs and pods and move their deletion to take place in the teardown.
Any ideas? @red-hat-storage/top-level-reviewers

Reply:
@ebenahar I'm not sure I understand the issue. Can't we just use the PVC factory fixture which handles the teardown of the resource for us?